lib_cxng/include/lcx_crc.h

```diff
@@ -60,6 +60,35 @@ uint16_t cx_crc16(const void *buffer, size_t len);
  */
 uint16_t cx_crc16_update(uint16_t crc, const void *buffer, size_t len);
 
+/** CRC32 initial value */
+#define CX_CRC32_INIT 0xFFFFFFFF
```

Comment:
Do we need this `define` here? it seems only used in the `c` code :thinking:  ?

Reply:
It's similar as what is done on CRC16, and I'm not sure it couldn't be used by app using `cx_crc32_update` rather than the more direct `cx_crc32`